package.json

```diff
@@ -1755,6 +1764,11 @@
           "when": "view == confluent-topics && viewItem =~ /ccloud-kafka-topic.*-flinkable.*/",
           "group": "z_openInCloud"
         },
+        {
+          "command": "confluent.artifacts.query",
+          "when": "view == confluent-flink-artifacts",
```

Comment:
We'll need to make sure this only applies to the "Artifacts" mode of the multi-view provider, since switching to UDF-mode currently also shows this action:

<img width="373" height="87" alt="Image" src="https://github.com/user-attachments/assets/60b028ac-c368-4153-bfc9-a77ebf50d775" />

```suggestion
          "when": "view == confluent-flink-artifacts && confluent.flinkArtifactsUDFsViewMode == artifacts",
```